src/components/AllQuests.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import getQuests from "../modules/getQuests";
+import QuestCard from "./QuestCard";
+import { Grid } from "semantic-ui-react";
+
+const AllQuests = () => {
+  const dispatch = useDispatch();
+  const quests = useSelector((state) => {
+    return state.quests.quests;
+  });
+
+  useEffect(() => {
+    getQuests(dispatch);
+  }, []);
+
+  const buildQuestCards = () => {
```

Comment:
I think this is really overcomplicating it.
If you just provide the grid with how many columns you want, and all the children in a list, it will automatically assign them slots and fill it row by row

Reply:
removed now